python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1368,27 +1376,28 @@ def annotate_dataframe(self, df, G, edge_vertex_col_names):
         -------
         A copy of df with additional columns corresponding to properties for
         the edge in the row.
-        FIXME: also provide the ability to annotate vertex data.
 
         Examples
         --------
         >>> import cugraph
         >>> import cudf
         >>> from cugraph.experimental import PropertyGraph
         >>> df = cudf.DataFrame(columns=["src", "dst", "some_property"],
-        ...                              data=[(99, 22, "a"),
-        ...                                    (98, 34, "b"),
-        ...                                    (97, 56, "c"),
-        ...                                    (96, 88, "d"),
-        ...                                  ])
+        ...                                       data=[(99, 22, "a"),
+        ...                                             (98, 34, "b"),
+        ...                                             (97, 56, "c"),
+        ...                                             (96, 88, "d"),
+        ...                                            ])
```

Comment:
indentation

I see many more indentation problems throughout the PR, so I won't list them all and will assume each will be fixed.

Reply:
fixed all of them